WebPlatform/server.alusus

```diff
@@ -77,14 +96,14 @@
             Spp.astMgr.insertAst(
                 ast {
                     if String.isEqual(method, "{{endpointMethod}}") && String.isEqual(uri, "{{endpointUri}}" ) {
-                        __funcName__(connection);
+                        fullref(connection);
                         return 1;
```

Comment:
what is fullref

Reply:
`fullref` gets replaced with the full qualifier of the function instead of just the function name, for example `MyModule.myFunction`.